### PR TITLE
added head-libs file every component

### PR DESCRIPTION
**Pull request by rohitgupta274509**
please review and approve it.

### CODE REVIEW

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html [hunk 1]
```diff
@@ -10,13 +10,13 @@
...
 <link href="https://unpkg.com/primevue/resources/themes/lara-light-indigo/theme.css" rel="stylesheet">
 <link href="https://unpkg.com/primevue/resources/primevue.min.css" rel="stylesheet">
 <link href="https://unpkg.com/primeicons/primeicons.css" rel="stylesheet">
-
 <link href="https://unpkg.com/primeflex@^3/primeflex.min.css" rel="stylesheet" />
 
-
 <script src="https://unpkg.com/vue@3/dist/vue.global.js"></script>
 <script src="https://unpkg.com/primevue@^3/core/core.min.js"></script>
 <script src="https://unpkg.com/primevue/button/button.min.js"></script>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html:22]: Move this to button headlibs

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html [hunk 2]
```diff
@@ -33,97 +33,74 @@
 <script src="https://unpkg.com/primevue@^3/skeleton/skeleton.min.js"></script>
 <script src="https://unpkg.com/primevue@^3/menubar/menubar.min.js"></script>
 <script src="https://unpkg.com/primevue@^3/tree/tree.min.js"></script>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html:35]: Move this to sidebar headlibs

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html [hunk 3]
```diff
@@ -33,97 +33,74 @@
...
-
-
 <script src="https://unpkg.com/primevue@^3/datatable/datatable.min.js"></script>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html:36]: Move this to datatable headlibs

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html [hunk 4]
```diff
@@ -33,97 +33,74 @@
...
+<sly data-sly-include="/apps/typerefinery/components/widgets/leafletMap/head-libs.html"></sly>
 
+<!-- Map Component -->
+<sly data-sly-include="/apps/typerefinery/components/widgets/map/head-libs.html"></sly>
+
+<!-- Ticker component-->
+<sly data-sly-include="/apps/typerefinery/components/widgets/ticker/head-libs.html"></sly>
 
 <!-- Chart JS CDN -->
 <script src="https://cdn.jsdelivr.net/npm/chart.js"></script>
```
Comment (Harikarthyk) [inline application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html:72]: move this to chart js

**Review by Harikarthyk — CHANGES_REQUESTED** [review]

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html (hunk 1 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html]: done

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html (hunk 2 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html]: done.

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html (hunk 3 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html]: done .

File: application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html (hunk 4 above)
Comment (rohitgupta274509) [reply application/backend/src/main/resources/apps/typerefinery/components/structure/page/head-libs.html]: done .

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**